content/en/docs/ambient/usage/verify-mtls-enabled/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Verify mutual TLS is enabled
+description: Understand how to verify mTLS is enabled among workloads in an ambient mesh.
+weight: 15
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+Once you have added applications to an ambient mesh, you can easily validate mTLS is enabled among your workloads in the ambient mesh, using one or more of the methods below:
+
+### Validate workload's ztunnel configurations
+
+Using the convenient `istioctl ztunnel-config workloads` command, you can view if your workload is configured to send and accept HBONE traffic via the value for the `PROTOCOL` column, for example:
+
+{{< text syntax=bash >}}
+$ istioctl ztunnel-config workloads
+NAMESPACE    POD NAME                                IP         NODE                     WAYPOINT PROTOCOL
+default      details-v1-857849f66-ft8wx              10.42.0.5  k3d-k3s-default-agent-0  None     HBONE
+default      kubernetes                              172.20.0.3                          None     TCP
+default      productpage-v1-c5b7f7dbc-hlhpd          10.42.0.8  k3d-k3s-default-agent-0  None     HBONE
+default      ratings-v1-68d5f5486b-b5sbj             10.42.0.6  k3d-k3s-default-agent-0  None     HBONE
+default      reviews-v1-7dc5fc4b46-ndrq9             10.42.1.5  k3d-k3s-default-agent-1  None     HBONE
+default      reviews-v2-6cf45d556b-4k4md             10.42.0.7  k3d-k3s-default-agent-0  None     HBONE
+default      reviews-v3-86cb7d97f8-zxzl4             10.42.1.6  k3d-k3s-default-agent-1  None     HBONE
+{{< /text >}}
+
+Having HBONE configured on your workload doesn't mean your workload will accept reject any plaintext traffic. If you want your workload to reject plaintext traffic, setting a `PeerAuthentication` policy with mTLS mode set to `STRICT` for your workload.
+
+### Validate mTLS from metrics
+
+If you have [installed Prometheus](docs/ops/integrations/prometheus/#installation), you can setup port-forwarding for Prometheus by using the following command:
```

Comment:
```suggestion
If you have [installed Prometheus](docs/ops/integrations/prometheus/#installation), you can setup port-forwarding by using the following command:
```

Reply:
This does more than just setting up the port-forward iirc. In other doc we say something like "opens the Prometheus UI" which it may make sense to stick with here as well.

Reply:
Updated, thanks!